### PR TITLE
Finish sampling on logout

### PR DESCRIPTION
**Pull request by anna1901**
https://trello.com/c/JfulUZxX

### CODE REVIEW

File: AirCasting/Utils/Bluetooth/NewBluetoothManager.swift [hunk 1]
```diff
@@ -374,6 +374,7 @@ final class NewBluetoothManager: NSObject, BluetoothCommunicator, CBCentralManag
     }
     
     func peripheral(_ peripheral: CBPeripheral, didUpdateValueFor characteristic: CBCharacteristic, error: Error?) {
+        Log.verbose("New value for characteristic: \(characteristic.value)")
```
Comment (jankrzempek) [inline AirCasting/Utils/Bluetooth/NewBluetoothManager.swift:377]: Do we need it at the end? If so maybe `Log.info`?

File: AirCasting/AuthViews/LogoutController.swift [hunk 2]
```diff
@@ -20,15 +23,23 @@ final class DefaultLogoutController: LogoutController {
...
-            DispatchQueue.main.async { self.removeDataController.removeData(); onEnd() }
+            DispatchQueue.main.async { self.end(onEnd: onEnd) }
         })
     }
+    
+    private func end(onEnd: @escaping () -> Void) {
+        removeDataController.removeData()
+        microphoneManager.stopRecording()
+        if let activeSessionUUID = activeSessionProvider.activeSession?.session.uuid {
+            bluetoothSessionRecorder.stopRecordingSession(with: activeSessionUUID, databaseChange: {_ in})
```
Comment (jankrzempek) [inline AirCasting/AuthViews/LogoutController.swift:41]: personal, but maybe `{ _ in }`? If no, please skip

**Review by jankrzempek — CHANGES_REQUESTED** [review]
Finishing the BT session works well, but the mic always crashes (tried few times). It crash just after `log out` button click.
Zrzut ekranu 2022-12-2 o 12 02 22

File: AirCasting/Utils/Bluetooth/NewBluetoothManager.swift (hunk 1 above)
Comment (anna1901) [reply AirCasting/Utils/Bluetooth/NewBluetoothManager.swift:377]: No, we don't need this. Thanks :)

**Review by jankrzempek — APPROVED** [review]
Sorry! 🫣

**Review by pawelgil — APPROVED** [review]

**Pull request merged**